app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -15,275 +15,289 @@
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/appBarLayout">
 
-            <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-                android:id="@+id/swipeRefreshLayout"
+            <androidx.coordinatorlayout.widget.CoordinatorLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
+                android:layout_height="match_parent">
 
-                <FrameLayout
+                <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+                    android:id="@+id/swipeRefreshLayout"
                     android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:nestedScrollingEnabled="true">
+                    android:layout_height="wrap_content">
 
-                    <androidx.constraintlayout.widget.ConstraintLayout
+                    <FrameLayout
                         android:layout_width="match_parent"
-                        android:layout_height="match_parent">
+                        android:layout_height="wrap_content"
+                        android:nestedScrollingEnabled="true">
 
-                        <TextView
-                            android:id="@+id/updateDownloadedText"
-                            style="@style/BodySmall"
-                            android:layout_width="0dp"
-                            android:layout_height="wrap_content"
-                            android:layout_marginHorizontal="@dimen/marginStandardMedium"
-                            android:layout_marginTop="@dimen/marginStandardMedium"
-                            android:text="@string/updateReadyTitle"
-                            app:layout_constraintEnd_toStartOf="@+id/installUpdate"
-                            app:layout_constraintStart_toStartOf="parent"
-                            app:layout_constraintTop_toTopOf="parent" />
-
-                        <com.google.android.material.button.MaterialButton
-                            android:id="@+id/installUpdate"
-                            style="@style/TextButton"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:text="@string/buttonInstall"
-                            app:icon="@drawable/ic_app_update"
-                            app:iconPadding="@dimen/marginStandardSmall"
-                            app:iconSize="@dimen/marginStandardMedium"
-                            app:layout_constraintEnd_toEndOf="parent"
-                            app:layout_constraintTop_toTopOf="parent" />
-
-                        <com.google.android.material.divider.MaterialDivider
-                            android:id="@+id/installUpdateDivider"
+                        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Maybe you can only have the `ConstraintLayout` here instead of a `ConstraintLayout` inside a `FrameLayout` ?

Reply:
This is simplified in a following PR : https://github.com/Infomaniak/android-kMail/pull/1579

Reply:
Noted, will be handled in a foreseeable future, since it's not tied to the Tablet mode feature.